src/reqs.js

```diff
@@ -2,7 +2,9 @@ const express = require('express');
 const path = require('path');
 const exphbs = require('express-handlebars');
 const fs = require('fs');
+const controllers = require('./controllers');
+const { serverError, notFound } = require('./controllers/error');
 
-module.exports = {
-  express, path, exphbs, fs,
-};
+console.log(serverError, notFound);
+
+module.exports = { express, path, exphbs, fs, controllers, serverError, notFound };
```

Comment:
`fs` And `notFound` are not ever imported after this export. @FarahZaqout Remember you're the one who wanted to be slick and make a `/reqs`.